src/test/resources/com/aws/greengrass/clientdevices/auth/config_with_ec_ca.yaml

```diff
@@ -2,10 +2,10 @@
 services:
   aws.greengrass.clientdevices.Auth:
     configuration:
-      ca_type: ["ECDSA_P256"]
+      certificateAuthority:
+        ca_type: ["ECDSA_P256"]
   main:
     lifecycle:
-      install:
-        all: echo All installed
```

Comment:
why is this changed? this isn't a valid recipe anymore.

Reply:
Not that it needs to be a valid recipe, you can just remove the echo thing.

Reply:
yeah, tests using invalid recipes have unnecessary component broken logs.